HW_7/controller.py

```diff
@@ -0,0 +1,29 @@
+import model
+import view
+from logger import log
+
+
+@log
+def start():
+    """Стартовая функция"""
+    view.greatings()
+    button = view.menu()
+
+    if button == 1:
+        view.print_book(model.get_data())
+        start()
+    elif button == 2:
+        model.add_data(view.add_record())
+        start()
+    elif button == 3:
+        model.add_data(view.editor(model.get_data_id(view.request_id())))
+        start()
+    elif button == 4:
+        view.print_book(model.get_data_last_name(view.request_last_name()))
+        start()
+    else:
+        print('Работа окончена')
+        exit
```

Comment:
Во-первых, эта строчка не нужно, программа и так завершиться. Во-вторых, эта строчка не работает, так как функция `exit` не вызывается. В-третьих, я говорил, что таким образом завершить работу программы не рекомендуется, но если очень хочется, то уж лучше делать так:
```suggestion
        sys.exit()
```